### PR TITLE
Fixed the account issue for page blob

### PR DESCRIPTION
**Pull request by sima-zhu**
Fixed the live tests.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=129733&view=logs&j=3f0b3bd8-2d88-59b8-ec21-d9a54670d8fb&t=d461725e-ef03-51a5-7040-91cc48a61aa8&l=486

### CODE REVIEW

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy [hunk 1]
```diff
@@ -164,6 +167,8 @@ class APISpec extends Specification {
         cc = primaryBlobServiceClient.getBlobContainerClient(containerName)
         ccAsync = primaryBlobServiceAsyncClient.getBlobContainerAsyncClient(containerName)
         cc.create()
+        ccPremium = premiumBlobServiceClient.getBlobContainerClient(containerName)
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy]: We should generate this for every test

Comment (sima-zhu) [reply sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy]: Just moved

**Comment by sima-zhu** [comment]
dev.azure.com/azure-sdk/public/_build/results?buildId=130162&view=logs&s=6884a131-87da-5381-61f3-d7acc3b91d76

**Comment by sima-zhu** [comment]
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=130166&view=logs&j=3f0b3bd8-2d88-59b8-ec21-d9a54670d8fb

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy [hunk 2]
```diff
@@ -61,6 +60,9 @@ class APISpec extends Specification {
     @Shared
     BlobContainerClient cc
 
+    @Shared
+    BlobContainerClient ccPremium
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy]: Don't need this globally shared property, can scope it to the one test

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ContainerAPITest.groovy [hunk 3]
```diff
@@ -602,9 +652,12 @@ class ContainerAPITest extends APISpec {
         blob.getProperties().getBlobSequenceNumber() == 0
         blob.getProperties().isServerEncrypted()
         blob.getProperties().isAccessTierInferred()
-        blob.getProperties().getAccessTier() == AccessTier.HOT
+        blob.getProperties().getAccessTier() == AccessTier.P10
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ContainerAPITest.groovy]: I'm guessing P10 is the default? It's determined by blob size right?

Comment (sima-zhu) [reply sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ContainerAPITest.groovy]: https://docs.microsoft.com/en-us/azure/virtual-machines/windows/disks-types#premium-ssd

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy (hunk 2 above)
Comment (sima-zhu) [reply sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy]: Did not see the comments here before merge. We can move it with some other changes. Sorry to miss your comment.

**Pull request merged**